examples/sampling/link_prediction.py

```diff
@@ -370,6 +370,8 @@ def train(args, device, g, reverse_eids, seed_edges, model, use_uva):
             loss.backward()
             opt.step()
             total_loss += loss.item()
+            if (it + 1) == args.early_stop:
```

Comment:
I would break it at L355.

Reply:
The `args.early_stop` parameter specifies the expected number of steps for each training epoch. Therefore, I would like to place it in this position, **as is done in the original DGL link prediction example**.